.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,47 @@
+name: pre-commit
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+    tags:
```

Comment:
Definitely shouldn't be happening on release right?

Reply:
Hmmm, maybe it should, it certainly won't help and everything will be the same then – and same is good

Reply:
As the guy that has to deal with releases, unless it's absolutely necessary please don't put stuff on tag deployment.

Reply:
yeah, removed it now – the comment was stale